dprint.json

```diff
@@ -7,6 +7,7 @@
   "excludes": [
     "**/node_modules",
     "**/ui/raidboss/data",
+    "**/ui/dps",
```

Comment:
I think it's not all of ui/dps, it's just `ui/dps/rdmty/dps.js`

Reply:
Oh, I'll just add a dprint ignore comment to this file in this case.